azure-quantum/azure/quantum/serialization/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .problem_pb2 import (
```

Comment:
Missing Copyright header.

Reply:
This is generated code. 
We should not edit it